sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/AnalyticsTest.java

```diff
@@ -87,14 +93,9 @@ public void notInit() {
         Analytics.trackEvent("test", new HashMap<String, String>());
         Analytics.trackPage("test");
         Analytics.trackPage("test", new HashMap<String, String>());
```

Comment:
Why removing those lines? The code should just simulate onStart with context before to restore those singleton tests.

Reply:
This isn't valid since this PR also fixes the bug that you mentioned. If it isn't initialized then `getTransmissionTarget` will return `null` which is covered by another unit test.

Reply:
I was talking about initializing.
This is the test that checks we always get same instance, I don't see any other test doing that, please add the onStart mock and restore the tests.

Reply:
Oh Github UI tricked me into thinking it was the singleton tests while its another test...